app/scripts/services/DataRepository.js

```diff
@@ -74,6 +77,29 @@ var DataRepository = function (recordSets) {
     return this.certifications;
   };
 
+  // Note: Audit logs are different to the other record sets because they are loaded "on demand", so this
+  // method must handle parsing the raw data first as necessary.
+  this.getAuditLogs = function getAuditLogs() {
+    if (this.auditLogs === null) {
+      // We haven't loaded the audit logs, so do it now.
+      this.auditLogs = [];
+      var self = this;
+      if (this.recordSets.hasOwnProperty(collectionNames.auditLog)) {
+        this.recordSets.auditLog.forEach(
+          function (auditLog) {
+            var auditLogHelper = new AuditLogHelper();
```

Comment:
Any reason not to use the instance of `AuditLogHelper` instantiated on line 27? Or can that not be accessed from here?

Reply:
Only that I completely forgot it was there!! Well spotted.

I've changed it now to use the instance that's created on line 27, and I'll merge the PR back into master.